From e06be392e42c23dd9bd7205445f3c5232ddb9ed7 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyfox@gentoo.org>
Date: Tue, 19 Nov 2013 15:17:52 +0300
Subject: [PATCH] modules/png-module.c: add declaration of 'memset'
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

 * QA Notice: Package triggers severe warnings which indicate that it
 *            may exhibit random runtime failures.
 * /tmp/portage/sys-libs/slang-2.2.4/work/slang-2.2.4/modules/png-module.c:122:2: warning: incompatible implicit declaration of built-in function ‘memset’ [enabled by default]

Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>
---
 modules/png-module.c  |   1 +
 3 files changed, 403 insertions(+), 259 deletions(-)

diff --git a/modules/png-module.c b/modules/png-module.c
index 92ede3d..fed9392 100644
--- a/modules/png-module.c
+++ b/modules/png-module.c
@@ -23,6 +23,7 @@ USA.
 #include "config.h"
 
 #include <stdio.h>
+#include <string.h> /* memset() */
 #include <errno.h>
 #include <slang.h>
 
-- 
1.8.4

